skipper.go

```diff
@@ -888,8 +892,13 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 	}
 	defer o.SecretsRegistry.Close()
 
+	sp := secrets.NewSecretPaths()
+	for _, p := range o.CredentialsPaths {
+		sp.Add(p)
```

Comment:
I think we should handle the error that `*fsnotify.Watcher.Add()` may return.

Reply:
not needed anymore